src/test/java/com/example/rets_api/repository/PropertyRepositoryJPATest.java

```diff
@@ -70,9 +74,27 @@ public void should_store_a_PropertyAndSchoolAndRoom() {
         assertEquals(propertySaved.getRoomList().get(0).getRoomType(), propertyToSave.getRoomList().get(0).getRoomType());
         assertEquals(propertySaved.getRoomList().get(1).getRoomType(), propertyToSave.getRoomList().get(1).getRoomType());
 
+        assertNotNull(propertySaved.getFinancialData());
+        assertEquals(propertySaved.getFinancialData().getLeaseIndicator(), propertyToSave.getFinancialData().getLeaseIndicator());
```

Comment:
You can use the method checkAllFields_FinancialData from FinancialDataConverterTest, instead replicate all code again